src/Search/Microsoft.Azure.Search.nuspec

```diff
@@ -30,9 +30,9 @@
   </metadata>
   <files>
     <file src="Search\**\*.cs" target="src" />
-    <file src="..\binaries\portable\Microsoft.Azure.Search.dll" target="lib\portable-net45+sl50+wp80+win" />
-    <file src="..\binaries\portable\Microsoft.Azure.Search.pdb" target="lib\portable-net45+sl50+wp80+win" />
-    <file src="..\binaries\portable\Microsoft.Azure.Search.xml" target="lib\portable-net45+sl50+wp80+win" />
+    <file src="..\binaries\portable\Microsoft.Azure.Search.dll" target="lib\portable-net45+wp8+wpa81+win" />
```

Comment:
this is in line now with rest libraries, but it is not the most accurate. Once my PR to get all library built for 4.5 merged, I will refresh all